llvm/test/CodeGen/DirectX/any.ll

```diff
@@ -0,0 +1,105 @@
+; RUN: opt -S -dxil-op-lower < %s | FileCheck %s
+
+; Make sure dxil operation function calls for any are generated for float and half.
+
+; CHECK:icmp ne i1 %{{.*}}, false
+define noundef i1 @any_bool(i1 noundef %p0) {
```

Comment:
Probably worth adding a `; CHECK-LABEL: define {{.*}} @any_bool` or so to make sure these checks are looking at the right functions and also potentially improve the error message if they fail